README.rst

```diff
@@ -1,46 +1,40 @@
-pip
-===
-
-The `PyPA recommended`_ tool for installing Python packages.
+pip - The Python Package Installer
+==================================
 
 .. image:: https://img.shields.io/pypi/v/pip.svg
    :target: https://pypi.org/project/pip/
 
-.. image:: https://img.shields.io/travis/pypa/pip/master.svg?label=travis-ci
-   :target: https://travis-ci.org/pypa/pip
-
-.. image:: https://img.shields.io/appveyor/ci/pypa/pip.svg?label=appveyor-ci
-   :target: https://ci.appveyor.com/project/pypa/pip/history
-
 .. image:: https://readthedocs.org/projects/pip/badge/?version=latest
    :target: https://pip.pypa.io/en/latest
 
-.. image:: https://dev.azure.com/pypa/pip/_apis/build/status/Linux?branchName=master&label=Windows
-   :target: https://dev.azure.com/pypa/pip/_build/latest?definitionId=6
+pip is the `package installer`_ for Python. You can use pip to install packages from the `Python Package Index`_ and other indexes.
 
-.. image:: https://dev.azure.com/pypa/pip/_apis/build/status/macOS?branchName=master&label=macOS
-   :target: https://dev.azure.com/pypa/pip/_build/latest?definitionId=7
-
-.. image:: https://dev.azure.com/pypa/pip/_apis/build/status/Linux?branchName=master&label=Linux
-   :target: https://dev.azure.com/pypa/pip/_build/latest?definitionId=4
+Please take a look at our documentation for how to install and use pip:
 
 * `Installation`_
 * `Documentation`_
 * `Changelog`_
-* `GitHub Page`_
+
+If you find bugs, need help, or want to talk to the developers please use our mailing lists or chat rooms:
+
 * `Issue Tracking`_
 * `User mailing list`_
```

Comment:
This links to a mailing list for python-virtualenv. Is that the right list?

Reply:
It was originally a shared list (long before PyPA even existed). That list is almost dead these days, but I don't know that there's a better one. Distutils-sig?

Reply:
That's the best (only?) one I can think of, but historically requests for help have only been for more advanced use cases rather than for beginners. I think that's probably because the [description says](https://www.python.org/community/sigs/current/distutils-sig/) the list is for "design, implementation, and maintenance" issues rather than general help. I don't know how list members would feel about publicizing it for more general help.

Reply:
I just remembered there is also the new Discourse category: https://discuss.python.org/c/packaging, but I can't speak to the intended scope since it was created only recently.

Reply:
I'll be happy for us to replace this and use Discourse.

That said, it's a separate change from the improvements here IMO and we shouldn't block these changes for that.

Reply:
Okay, then I would recommend removing the reference to the virtualenv mailing list at least. Distutils-SIG and/or Discourse can be added later.

Reply:
#6214